scripts/collect-sboms-from-maven-central.py

```diff
@@ -146,6 +146,6 @@ def is_sbom(name):
         if sboms:
             sbom = sboms[0]
             url = f"{project.url()}/{lastVersion}/{sbom}"
-            cache = f'{pmc}/maven/{project.group_id}/{project.artifact_id}/{lastVersion}/{sbom}'
+            cache = f'{pmc}/maven-central/{project.group_id}/{project.artifact_id}/{lastVersion}/{sbom}'
```

Comment:
I think we should keep `maven` here: I modeled the `sboms` directory structure after https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst

Reply:
I understand: it depends on the meaning we want to give to this
IMHO, purl type of some components is not the best value: for example, Airflow has pypi AND npm, and i the future, mixing multiple purl type will happen more and more (image Docker images)

that's why I changed in fact the meaning to "location where the initial storage has been done"
but I confirm that later, I want to have a human description of SBOMs, to let users know how many components are described of which type

Reply:
(very useful discussion: yes, this study opens key topics of interest)

Reply:
I think it is more meaningful to use the "identification of the artifact" rather than the "location where the initial storage has been done" - after all, an SBOM describes an artifact, so we should have some way to refer to those. I don't think that prevents creating human description of SBOMs, which I agree is also useful.

Reply:
> I think it is more meaningful to use the "identification of the artifact" rather than the "location where the initial storage has been done"

I agree to the idea
the reality is that Airflow content is listed under a `pypi` directory, but SBOMs describe either `npm` or `application` (see metadata https://github.com/apache/security-site/blob/sboms/sboms/airflow/airflow-2.10.2.md )

given there are multiple artifacts, it's not so easy to say what artifacts are described: the only clear fact is "where did we get these SBOMs from", and when writing `dlcdn` and `website` in additon to `maven-central`, I was even myself surprised about the value I got

> I don't think that prevents creating human description of SBOMs, which I agree is also useful.

yes, thanks for merging #25 , this is a true discovery to me to see what other projects did

Reply:
> > I think it is more meaningful to use the "identification of the artifact" rather than the "location where the initial storage has been done"
> 
> I agree to the idea
> the reality is that Airflow content is listed under a `pypi` directory, but SBOMs describe either `npm` or `application` (see metadata https://github.com/apache/security-site/blob/sboms/sboms/airflow/airflow-2.10.2.md )
>
> given there are multiple artifacts, it's not so easy to say what artifacts are described

Sure - but then I think we should fix _that_

> the only clear fact is "where did we get these SBOMs from" (aka. "where did the project publish its SBOMs?"), and when writing `dlcdn` and `website` in additon to `maven-central`, I was even myself surprised about the values I got: clarifying where projects publish their SBOMs is more valuable than I thought

Yeah, I think we should probably still do that, perhaps in `scripts/sbom-locations.json` or something that's to evolve from that.